sources/osg/FrameBufferObject.js

```diff
@@ -285,6 +289,14 @@ FrameBufferObject.prototype = MACROUTILS.objectInherit( GLObject.prototype, MACR
 
                 this.bindFrameBufferObject();
 
+
+                // Check extDrawBuffers extension
+                var extDrawBuffers = this._extDrawBuffers;
+                if ( extDrawBuffers === undefined ) { // will be null if not supported
+                    extDrawBuffers = gl.getExtension( 'WEBGL_draw_buffers' );
```

Comment:
should be using webglcaps to get extension.
(as extension could be filtered out)